extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -1,14 +1,16 @@
 <?php
 class WikiaRssExternalController extends WikiaController {
 
+	const CACHE_TIME = 3600;
```

Comment:
It was always 1 hour. We will now have two caches - Memcache and Varnish. If we use cacheWithLock it doubles the maximum TTL (up to 2 hours). I think this is a sensible maximum for an RSS feed. If we change this value to 3 hours then it would mean that the data can be 6 hours old.

```
Returns cached data if possible
if cached data is older than $cacheTime but fresher than twice that time
- first thread to request it will start getting data
- other threads in the meantime will continue receiving old data
```